components/console-backend-service/internal/domain/assetstore/assetstore_retriever.go

```diff
@@ -0,0 +1,26 @@
+package assetstore
```

Comment:
Please read guidelines (contribution rules) for the project.

I know that separating the Retriever is cleaner, but currently we force some rules regarding file naming in domain. However, I like this idea. We need to think about it. Maybe you could include this file in contribution rules?